src/components/chart/IndicatorOption.vue

```diff
@@ -1,76 +1,68 @@
 <template>
-  <div
-    class="indicator-option form-group"
-    :class="['-' + type, inline && 'indicator-option--inline']"
-  >
-    <label>
-      {{ label }}
-    </label>
-
-    <dropdown-button
-      v-if="name === 'lineType'"
-      v-model="value"
-      :options="{ 0: 'Simple', 1: 'with steps' }"
-      class="-outline form-control -arrow"
-      placeholder="lineType"
-      @input="setValue($event)"
-    ></dropdown-button>
-    <dropdown-button
-      v-else-if="/linestyle$/i.test(name)"
-      v-model="value"
-      :options="{
-        0: 'Solid',
-        1: 'Dotted',
-        2: 'Dashed',
-        3: 'LargeDashed',
-        4: 'SparseDotted'
-      }"
-      class="-outline form-control -arrow"
-      placeholder="lineStyle"
-      @input="setValue($event)"
-    ></dropdown-button>
-    <color-picker-control
-      v-else-if="type === 'color'"
+  <div class="indicator-option">
+    <component
+      v-if="type"
+      :is="componentName"
+      :pane-id="paneId"
+      :indicator-id="indicatorId"
       :label="label"
-      model="rgb"
-      allow-null
-      :value="value"
-      @input="setValue($event)"
-      @close="reloadIndicator"
-    ></color-picker-control>
-    <editable
-      v-else-if="type === 'string' || type === 'number'"
-      class="form-control"
       :value="value"
+      :definition="definition"
       @input="setValue($event)"
-    ></editable>
-    <label v-else-if="type === 'boolean'" class="checkbox-control">
-      <input
-        type="checkbox"
-        class="form-control"
-        :checked="value"
-        @change="setValue($event.target.checked)"
-      />
-      <span>{{ label }}</span>
-      <div></div>
-    </label>
+    >
+      <template v-if="definition.description" #description>
+        <i class="icon-info pl4" v-tippy :title="definition.description"></i>
+      </template>
+    </component>
   </div>
 </template>
 <script lang="ts">
 import { Component, Vue, Watch } from 'vue-property-decorator'
 import {
-  getDefaultIndicatorOptionValue,
-  getIndicatorOptionType
+  getIndicatorOptionType,
+  getDefaultIndicatorOptionValue
 } from './options'
-
-import DropdownButton from '@/components/framework/DropdownButton.vue'
-import ColorPickerControl from '../framework/picker/ColorPickerControl.vue'
+import { ALLOWED_OPTION_TYPES } from './serieBuilder'
 
 @Component({
-  name: 'IndicatorOption',
+  name: 'IndicatorOptions',
   components: {
-    DropdownButton,
-    ColorPickerControl
+    IndicatorOptionNumber: () =>
+      import(
+        /* webpackChunkName: "IndicatorOptionNumber"*/ '@/components/chart/options/IndicatorOptionNumber.vue'
```

Comment:
since we moved to Vite, isn't this comment obsolete?

Reply:
Interesting! I didn't knew that. will remove all webpackChunkName from the codebase in this PR.